caddyhttp/rewrite/to.go

```diff
@@ -18,8 +18,15 @@ func To(fs http.FileSystem, r *http.Request, to string, replacer httpserver.Repl
 
 	// try each rewrite paths
 	t := ""
+	query := ""
 	for _, v := range tos {
-		t = path.Clean(replacer.Replace(v))
+		t = replacer.Replace(v)
+		tparts := strings.Split(t, "?")
```

Comment:
Not really sure? I think more about possible cases and other solution to get the query.
And we can't really use url.Parse() to solve that problem.